src/components/Reports/GDPRPermissions/SubscriberGdpr.styles.js

```diff
@@ -1,8 +1,15 @@
 import styled from 'styled-components';
 import { fonts } from '../../styles/fonts';
 
-export const TextColumn = styled.span`
+const TextColumn = styled.span`
   p {
     font-size: ${fonts.fontLvl3};
   }
 `;
+
+const EmptyBox = styled.div`
+  padding: 48px;
```

Comment:
I think we have variables for in styled-components inside webapp

Reply:
Yes [here](https://github.com/FromDoppler/doppler-webapp/blob/master/src/components/styles/spacings.js#L11)

Reply:
Thanks!